docker-compose.yml

```diff
@@ -38,6 +38,10 @@ services:
       - type: bind
         source: /dev
         target: /dev
+      # Set local ~/.gitconfig to system git config within container. 
+      # If you don't have a ~/.gitconfig, please create one.
+      - $HOME/.gitconfig:/etc/gitconfig
```

Comment:
Do you know of any way for this to fail gracefully (skip it) if someone doesn't have this directory created?

Reply:
After some thought, I think it'd be best if just didn't use git for version control purposes from within the container (and limit it exclusively to the host). Just too much to configure (SSH credentials, file permissions, etc).